common/smoke-test/app.ts

```diff
@@ -5,7 +5,8 @@
 import { KeyVaultSecrets } from "./KeyVault";
 import { EventHubs } from "./EventHub";
 import { BlobStorage } from "./BlobStorage";
-import { CosmosDB } from "./CosmosDB";
+// Temporarily disabling cosmos db testing
```

Comment:
Should we file an issue to re-enable?

Reply:
Yes! https://github.com/Azure/azure-sdk-for-js/issues/8319